sunbird-mentoring/constants/routes.js

```diff
@@ -1522,6 +1522,920 @@ module.exports = {
+			targetRoute: {
+				path: '/user/v1/notification/template',
+				type: 'PATCH',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/account/search',
+			type: 'POST',
+			targetRoute: {
+				path: '/user/v1/account/search',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/permissions/create',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/permissions/create',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/permissions/update/:id',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/permissions/update/:id',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/permissions/list',
+			type: 'GET',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/permissions/list',
+				type: 'GET',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/permissions/delete/:id',
+			type: 'DELETE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/permissions/delete/:id',
+				type: 'DELETE',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/modules/create',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/modules/create',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/modules/update/:id',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/modules/update/:id',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/modules/list',
+			type: 'GET',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/modules/list',
+				type: 'GET',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/modules/delete/:id',
+			type: 'DELETE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/modules/delete/:id',
+				type: 'DELETE',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/role-permission-mapping/delete/:role_id',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/role-permission-mapping/delete/:role_id',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/role-permission-mapping/create/:role_id',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/role-permission-mapping/create/:role_id',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/role-permission-mapping/list',
+			type: 'POST',
+			priority: 'MUST_HAVE',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/role-permission-mapping/list',
+				type: 'POST',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/org-admin/updateUser',
```

Comment:
@rakeshSgr Let's remove all routes which are not required in sunbird like login, update, change password etc, only keep the necessary ones.

Reply:
resolved